### DISCUSSION TITLE
block  *about:addons* moz-extension://*?

### DISCUSSION THREAD

**Discussion by doublethefish** [proginosko/LeechBlockNG#504] [Q&A] [answered]
I find that some times I just disable the plugin. If it was also impossible to disable the plugin during that time, that would be awesome. That is block/limit
- *about:addons*
- moz-extension://*

**Comment by proginosko** [comment, marked as answer]
There is an option "Prevent access to **about:addons** at times when these sites are blocked" under Advanced Options.

If you want to block **moz-extension://*** you can use regular expression blocking under Advanced Options. Use this regex:

`^moz-extension:\/\/`

**Reply by doublethefish** [reply]
> There is an option "Prevent access to about:addons at times when these sites are blocked" under Advanced Options.

I had missed that there were two levels of advanced options! Thanks so much, that is **exactly** what I was hoping for!  👍🤘

> ^moz-extension:\/\/

Unless I missed something this (and other regex-y variants) didn't work. I assume because `moz-extension://` is a protocol or treated as one.

**Reply by proginosko** [reply]
I just tested a regex to block LeechBlock's Lockdown page and it worked fine. You probably shouldn't block all moz-extension URLs, though, because then you would end up blocking LeechBlock's blocking page (if that's what you're using) and get caught in a never-ending loop!

**Reply by doublethefish** [reply]
I must have tried the regex in the wrong place I guess.
